plugins/vars/sops.py

```diff
@@ -93,7 +93,9 @@ def get_vars(self, loader, path, entities, cache=True):
                         if os.path.exists(b_opath):
                             if os.path.isdir(b_opath):
                                 self._display.debug("\tprocessing dir %s" % opath)
-                                found_files = loader.find_vars_files(opath, entity.name, extensions=DEFAULT_VALID_EXTENSIONS)
+                                found_files = loader.find_vars_files(opath, entity.name)
+                                found_files = [file_path for file_path in found_files
+                                    		if any(file_path.endswith(extension) for extension in DEFAULT_VALID_EXTENSIONS)]
```

Comment:
Don't use tabs, and make sure the `if` is exactly below the `file_path`, not below the `[`.

Reply:
Yeah my editors being weird. Will fix now.